examples/asr/librispeech_emformer_rnnt/pipeline_demo.py

```diff
@@ -0,0 +1,50 @@
+from argparse import ArgumentParser
+import pathlib
+import torch
+import torchaudio
+from torchaudio.prototype.rnnt_pipeline import EMFORMER_RNNT_BASE_LIBRISPEECH
+
+
+SAMPLES_PER_CHUNK = 640
```

Comment:
Is this configureable at inference time? If it's tied to the way model is trained, I think it should be retrievable from `RNNTBundle`.